marshaling_test.go

```diff
@@ -211,7 +211,7 @@ func (s *Zuite) TestStructScan_notOptionalYetUndefined() {
 		Text string `ws:"text"`
 	}
 	err := ws.StructScan(&data)
-	require.EqualError(s.T(), err, "field text to struct field Text: undefined into not nullable")
+	require.EqualError(s.T(), err, "field text to struct field Text: cannot convert text to string, dest must be a ptr")
```

Comment:
How about `cannot convert undefined value to string, dest must be a *string`?

What tripped me is that `text` can be converted to `string`, but here we have an `undefined`.